jekyll/_includes/global/message-bar.md

```diff
@@ -0,0 +1,26 @@
+{% case include.type %}
+  {% when 'Warning' %}
+
+  <span class="warning" markdown="1">
+  ![Excamation Mark in a Triangle.](../images/global/warning-16.svg)
```

Comment:
All images share the same background styles but it shouldn't be the case for the SVGs that we use in the message bars 🙂

Reply:
Fixed in the next branch.